tasks/coverage.py

```diff
@@ -176,12 +177,51 @@ def upload_coverage_to_s3(ctx: Context):
     print(color_message(f'Successfully removed the local {COV_ARCHIVE_NAME}', Color.GREEN))
 
 
-def apply_missing_coverage(ctx: Context, from_commit_sha: str, debug: bool = False):
+def _get_dev_coverage_files(dev_cov_lines: str) -> set[str]:
+    browsed_dev_files = set()
+    for line in dev_cov_lines:
+        file_path = line.split(':')[0]
+        browsed_dev_files.add(file_path)
+    return browsed_dev_files
+
+
+def _merge_dev_in_main_coverage(main_cov_file: str, dev_cov_file: str) -> None:
+    with open(main_cov_file, encoding='utf-8') as main_cov:
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I'm not sure the encoding arg is useful, is it?

Reply:
The linter complains if I don't use it :') https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/unspecified-encoding.html